Assets/_Code/Data/Job/JobTask.cs

```diff
@@ -11,6 +11,8 @@ public class JobTask
         public ushort Index;
         
         public TextId LabelId;
+        public int TaskComplexity;
```

Comment:
i don't think these two fields need to be mirrored from the editor task data. they're purely for DBExport, right? If you need to look them up, you should look them up from the editor task list.